### DISCUSSION TITLE
How to build an api rest with phalcon

### DISCUSSION THREAD

**Discussion by Umoja7** [phalcon/cphalcon#16600] [General]
Hello, my name is Patrick, 
Im a new on the phalcon dev and I wanna to build an Api restfull with phalcon on my macos.
the problem, there no more tuto about phalcon api even how to install phalcon on macos.
can you please assist with

**Comment by niden** [comment]
Have a look at these resources

https://docs.phalcon.io/latest/installation/#macosx-using-brew

https://docs.phalcon.io/latest/tutorial-rest/
